### ISSUE TITLE
User guide

### ISSUE THREAD

**Issue by marco99asr** [CiscoDevNet/Glance#4]
Hi everyone,
im trying use Glance but i dont know where start.
Is there an user guide ?

**Comment by marco99asr** [comment]
im trying to change the default map but i dont kow wt i have to do.

**Comment by kennych** [comment]
hi @marco99asr,
Here are some simple guide for admin console:

There is console http://your_host/settings.html (http://your_host/#admin)

The default. account: 
[default account :http://your_host/settings.html](http://your_host/settings.html)

Some Sample maps are in svg format:
https://github.com/CiscoDevNet/Glance/tree/master/public/map

To change map:
open admin console page --- 1) settings  page (http://your_host/#admin )  -> 2) Register Map    -> 3) CMX Setup (changes the map name that created in Register Map) -> 4) Refresh the page will load the updated map.

The setting  page -  http://your_host/settings.html   has some basic setting options for system.

BR

**Comment by marco99asr** [comment]
To upload a new map, wt kind of files i need ?

**Comment by kennych** [comment]
Here is info about the map file and map mask files:
For example (you can reference the files under https://github.com/CiscoDevNet/Glance/tree/master/public/map):
normal-glance.svg for map, and normal-glance-mask.png, normal-glance-mask-half.png, and normal-glance-mask-quarter.png for map masks (theses mask files are used for different routines selection)

**Comment by marco99asr** [comment]
after map creation can i upload it in http://192.168.0.129/admin.html#map-register?

**Comment by kennych** [comment]
YES, you can change the map name (you registered via map registration) of CMX setup's map options, after you  have uploaded svg map file and map mask files with map name that you entered ). 
BTW:
     Map registration only upload the map relevant files to system (you can register multi-maps with different map names) and you can to change map name via "CMX Setup" of  map name option to enable the map.

**Comment by marco99asr** [comment]
After the upload in cmx Setup i see only 1 floor
![image](https://github.com/CiscoDevNet/Glance/assets/71874762/50a8ee96-2110-4519-b6d0-8c92efc3a409)

When i try to upload the map i don't receive any error/message 
The new map's name is custom, i use an svg map, png mask, same png mask-half and same png mask-quarter

**Comment by marco99asr** [comment]
{"status":"failure","message":"You are not authorized for this action!","version":"1.0.0.0"}
i receive this if i try to edit map name with "custom" in default floor "Glance's Store"

**Comment by kennych** [comment]
I'm not sure there is a defect or not, you can login with admin console with default admin account again.
and go to CMX setup page have a retry. it should support add multi-floors.

**Comment by kennych** [comment]
hi @marco99asr,
There is the API list at https://github.com/CiscoDevNet/Glance/blob/master/conf/routes
The simple flow to add map's campus, building, and floor is:
POST           /api/v1/campuses                                     controllers.glance.TrackCampus.add
POST           /api/v1/buildings                                       controllers.glance.TrackBuilding.add
POST          /api/v1/floors                                             controllers.glance.TrackerFloor.add

You can reference the code to setup your own map campuses, buildings, floors via API calls.
Hope this can help.

**Comment by kennych** [comment]
Duo to the Glance project has not under maintain for long time, some UX stuff may have some changes since the project committed.

**Comment by kennych** [comment]
Some basic info about how Glance work:
Maps -> Campuses -> Buildings -> Floors -> floors map & CMX floor that to be tracked
Users/Devices setup -> register the tracking device's MAC address or BLE device Mac Address
Map Editor UI ->  There is simple Map Editor UI to generate SVG format floor map
CMX/Meraki Callback settings -> Setup CMX/Meraki callback APIs path for CMX System or Meraki POST URI to receive location information from CMX/Meraki system
Admin UI -> Some simple admin console UI for setup basic information --Company Name, logo, floor accuracy calibration  setting etc.
WebSocket: Devices(People) location information updates pushing.
